content/en/docs/tasks/tools/included/kubectl_installs_linux.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "linux kubectl install"
+description: "Content for the Linux tab in the main page."
+toc_hide: true
+---
```

Comment:
This also looks like it might be intended to work like a Hugo [headless bundle](https://gohugo.io/content-management/page-bundles/#headless-bundle).

Reply:
@sftim thanks for this, a new one for me.  This would allow the pages to be included as snippets, but would mean you can't navigate to them at e.g. `https://kubernetes.io/docs/tasks/tools/included/kubectl_installs_linux` correct?

Reply:
That's the idea. Although, I like @kbhawkey's suggestions in https://github.com/kubernetes/website/pull/26682#discussion_r585091797 even more.

Reply:
Great, thanks - I do too.

Reply:
Both were done in https://github.com/kubernetes/website/pull/26682/commits/62ffd9d9a4e0fa6bac949ee8dfcfe12130afb357